generic/htx_test.py

```diff
@@ -76,7 +76,7 @@ def setup_htx(self):
             packages.extend(['libncurses5', 'g++',
                              'ncurses-dev', 'libncurses-dev'])
         elif detected_distro.name == 'SuSE':
-            packages.extend(['libncurses5', 'gcc-c++', 'ncurses-devel', 'tar'])
+            packages.extend(['libncurses6', 'gcc-c++', 'ncurses-devel', 'tar'])
```

Comment:
`libncurses5` is still available in Sles 15SP3 under the legacy module. Also this change causes failures on Sles15sp2, so make sure you handle this if you still want to pursue this.

Reply:
Harish I tested with Sles15sp2 also and it is working fine with libncurses6
localhost:/home/siri/avocado-fvt-wrapper/tests/avocado-misc-tests/generic # avocado run htx_test.py
JOB ID     : 5ba75e2ff61783ca263bacf61ef0572adec3b23e
JOB LOG    : /home/siri/avocado-fvt-wrapper/results/job-2021-02-25T16.20-5ba75e2/job.log
 (1/3) htx_test.py:HtxTest.test_start: PASS (53.02 s)
 (2/3) htx_test.py:HtxTest.test_check: PASS (121.72 s)
 (3/3) htx_test.py:HtxTest.test_stop: PASS (7.86 s)
RESULTS    : PASS 3 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /home/siri/avocado-fvt-wrapper/results/job-2021-02-25T16.20-5ba75e2/results.html
JOB TIME   : 183.07 s
localhost:/home/siri/avocado-fvt-wrapper/tests/avocado-misc-tests/generic #

Reply:
@shirishaganta Sorry, I must have misguided you with `Sles15sp2` but as a matter of fact I intended to handle other SLES version like Sles12 for instance.

Reply:
on sles12sp5 also it worked fine
Sles12sp5:
linux-7ve8:/home/siri/avocado-fvt-wrapper/tests/avocado-misc-tests/generic # avocado run htx_test.py
JOB ID : ee10da72c9910281cfb5a337b3ad41cd85b5c3af
JOB LOG : /home/siri/avocado-fvt-wrapper/results/job-2021-02-25T01.11-ee10da7/job.log
(1/3) htx_test.py:HtxTest.test_start: PASS (35.28 s)
(2/3) htx_test.py:HtxTest.test_check: PASS (120.80 s)
(3/3) htx_test.py:HtxTest.test_stop: PASS (5.99 s)
RESULTS : PASS 3 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML : /home/siri/avocado-fvt-wrapper/results/job-2021-02-25T01.11-ee10da7/results.html
JOB TIME : 162.97 s
linux-7ve8:/home/siri/avocado-fvt-wrapper/tests/avocado-misc-tests/generic #

Reply:
Right, verified that the changes wont affect distros starting SLES12SP2 which provides the package **only** in the maintenance updates and not the GA's.

While this change is valid what is the need for this change considering the following?
> libncurses5 is still available in Sles 15SP3 under the legacy module

Reply:
After configuring legacy module repo it is working fine. So closing this PR. Sorry for the incovenience